pkgs/top-level/python-packages.nix

```
@@ -8254,10 +8254,7 @@ self: super: with self; {
 
   nfcpy = callPackage ../development/python-modules/nfcpy { };
 
-  nftables = toPythonModule (pkgs.nftables.override {
-    python3 = python;
-    withPython = true;
-  });
+  nftables = callPackage ../os-specific/linux/nftables/python.nix { nftables = pkgs.nftables; };
```

```suggestion
  nftables = callPackage ../os-specific/linux/nftables/python.nix {
    inherit (pkgs) nftables;
  };
```
seems to be the more common pattern in this file